### PR TITLE
Fix pooling when the operator is None and the input is 1d

### PR DESCRIPTION
**Pull request by ermanok**

### CODE REVIEW

File: izer/izer.py [hunk 1]
```diff
@@ -502,7 +502,8 @@ def main():
                            f'{pixelcount}) does not match the '
                            f'sum of all pixels ({pixels}) in the layer\'s `in_sequences`.')
                 input_dim[ll] = conf_input_dim[ll]
-        if operator[ll] != op.CONV1D:
+
+        if not (operator[ll] == op.CONV1D or (operator[ll] == op.NONE and input_dim[ll][1] == 1)):
```
Comment (rotx-maxim) [inline izer/izer.py:506]: if operator[ll] != op.CONV1D and (operator[ll] != op.NONE or input_dim[ll][1] != 1):

File: izer/izer.py [hunk 2]
```diff
@@ -521,7 +523,7 @@ def main():
             eprint(f'{layer_pfx(ll)}Pooling or zero-padding results in a zero data '
                    f'dimension (input {input_dim[ll]}, result {pooled_dim[ll]}).')
 
-        if operator[ll] != op.CONV1D:
+        if not (operator[ll] == op.CONV1D or (operator[ll] == op.NONE and input_dim[ll][1] == 1)):
```
Comment (rotx-maxim) [inline izer/izer.py:526]: if operator[ll] != op.CONV1D and (operator[ll] != op.NONE or input_dim[ll][1] != 1):

**Review by rotx-maxim — COMMENTED** [review]
Can we simplify the logic?

File: izer/izer.py (hunk 1 above)
Comment (ermanok) [reply izer/izer.py]: Changed

File: izer/izer.py (hunk 2 above)
Comment (ermanok) [reply izer/izer.py]: changed

**Review by rotx-maxim — APPROVED** [review]

**Pull request merged**